lib/Parser/ParseFlags.h

```diff
@@ -51,6 +51,9 @@ enum
                                      //  let/const in global scope instead of eval scope so that they can be preserved across console inputs
     fscrNoAsmJs = 1 << 25, // Disable generation of asm.js code
     fscrIsModuleCode = 1 << 26, // Current code should be parsed as a module body
-    fscrAll = (1 << 27) - 1
+
+    fscrIndirect = 1 << 27,
```

Comment:
In that case I may remove 'fscrIndirect', rename 'fscrEval' to 'fscrDirectEval', and revert changes to EvalHelperAPI.

---
In reply to: [73578663](https://github.com/Microsoft/ChakraCore/pull/1377#discussion_r73578663) [](ancestors = 73578663)